tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Kinesis/AwsKinesisCommon.cs

```diff
@@ -38,6 +38,17 @@ internal static class AwsKinesisCommon
             return arnComponents[1];
         }
 
+        public static string? GetStreamName(IAmazonKinesisRequestWithStreamNameAndStreamArn request)
+        {
+            string? streamName = StreamNameFromARN(request.StreamARN);
+            if (string.IsNullOrEmpty(streamName))
```

Comment:
```suggestion
            if (StringUtil.IsNullOrEmpty(streamName))
```

Small nitpick, we added this recently (used in `StreamNameFromARN`) so it may not be on your branch unless you've rebased, but this `StringUtil.IsNullOrEmpty` allows for `string.IsNullOrEmpty` to work with `#nullable enable` as otherwise for older versions of .NET if you assert that a `string.IsNullOrEmpty` is `false` (e.g., it has a value) it will still show up as "possibly null".

This just gets around that, not very important though.